src/init/ossec-server.sh

```diff
@@ -131,25 +131,29 @@ help()
 }
 
 AUTHD_MSG="This option is deprecated because Authd is now enabled by default. If you want to change it, modify the ossec.conf file."
+DATABASE_MSG="This option is deprecated because Database is now enabled by default."
```

Comment:
_This option is deprecated because **the database output** is now enabled by default._